api/envoy/service/auth/v2/attribute_context.proto

```diff
@@ -102,6 +103,9 @@ message AttributeContext {
     // The network protocol used with the request, such as
     // "http/1.1", "spdy/3", "h2", "h2c"
     string protocol = 10;
+
+    // The HTTP request body.
+    envoy.api.v2.core.DataSource body = 11;
```

Comment:
not sure DataSource is appropriate here, pointing to a file doesn't make sense unless authz server are on the same box. `bytes` or `string` is enough?

Reply:
👍